quint/README.md

```diff
@@ -5,7 +5,7 @@ with [the Quint specification language](https://github.com/informalsystems/quint
 
 ## Dependencies
 
-- Node.js >= 14
+- Node.js >= 18
```

Comment:
Is it a common version of nodejs?

Reply:
Yes! This is the currently active LTS version. This addresses #867. See https://nodejs.dev/en/about/releases/

14 is already beyond its maintenance window.